src/app/tests/suites/TestGroupKeyManagementCluster.yaml

```diff
@@ -74,6 +74,345 @@ tests:
           constraints:
               minValue: 3
 
+    # TODO: Re-enabled when CI wouldn't break on this syntax
```

Comment:
Please file an issue to track this.  This should be able to happen right after TE2...

Reply:
Done.